packages/connectors/tsup.config.js

```diff
@@ -4,18 +4,20 @@ import { defineConfig } from 'tsup';
 export default defineConfig((options) => ({
   ...baseConfig(options, { withReact: false }),
   entry: ['src/index.ts'],
-  external: ['fuels', 'json-rpc-2.0'],
+  external: ['fuels', 'json-rpc-2.0', 'socket.io-client', 'axios'],
```

Comment:
```suggestion
  external: ['fuels', 'json-rpc-2.0'],
```

With the other changes done, this change should be reverted.